nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -179,6 +180,7 @@ bool BtActionServer<ActionT>::loadBehaviorTree(const std::string & bt_xml_filena
     return false;
   }
 
+  logger_file_ = std::make_unique<BT::FileLogger>(tree_, "bt_trace.fbl");
```

Comment:
A few questions:
- Should this be timestamped or somehow associated with a request? E.g. should we append to the end of the base name the timestamp so we don't override previous ones or maybe use the goal ID hash?
- Should this be optional so that the comment above doesn't generate a ton of files in the users' running directory, or allow the user to specify a directory to save these in?